Tests/Common/Orders/Fills/FutureFillModelTests.cs

```diff
@@ -64,6 +64,49 @@ public void Setup()
             Assert.AreEqual(OrderStatus.Filled, fill.Status);
         }
 
+        [Test]
+        public void PerformsStopMarketFill(
+            [Values] bool extendedMarketHours,
+            [Values] bool isInternal,
+            [Values(OrderDirection.Buy, OrderDirection.Sell)] OrderDirection orderDirection)
+        {
+            var symbol = Symbols.ES_Future_Chain;
+            var model = new FutureFillModel();
+            var quantity = orderDirection == OrderDirection.Buy ? 100 : -100;
+            var marketPrice = orderDirection == OrderDirection.Buy ? 102m : 101m;
+
+            var time = Noon.AddHours(-12);
+
+            var order = new StopMarketOrder(symbol, quantity, 101.124m, time);
+            var config = CreateTradeBarConfig(symbol, isInternal, extendedMarketHours);
+            var security = GetSecurity(config);
+            TimeKeeper.GetLocalTimeKeeper(TimeZones.NewYork).UpdateTime(time.ConvertToUtc(TimeZones.NewYork));
+            security.SetLocalTimeKeeper(TimeKeeper.GetLocalTimeKeeper(TimeZones.NewYork));
+            security.SetMarketPrice(new IndicatorDataPoint(symbol, time, marketPrice));
+
+            var fill = model.Fill(new FillModelParameters(
+                security,
+                order,
+                new MockSubscriptionDataConfigProvider(config),
+                Time.OneHour,
+                null)).Single();
+
+            if (extendedMarketHours && isInternal != true)
```

Comment:
`extendedMarketHours && !isInternal`?

Reply:
I have removed `isInternal` flag. It is not make sense.